src/runtime/dom-particle-base.ts

```diff
@@ -53,7 +54,7 @@ export class DomParticleBase extends Particle {
   /**
    * Override to return a dictionary to map into the template.
    */
-  render(stateArgs?): {} {
+  render(stateArgs?): Model {
```

Comment:
Model is used by CRDT and other classes, can we disambiguate this?

Reply:
Good idea. 
What's a better name? `ViewModel`?

Reply:
`ViewModel` could be apt, depending on notions of `View`. Usually I call this the `RenderModel` (since it's the result of `render`).

Reply:
I like `ViewModel` since it's a term-of-art in the industry. Is it possible that we will eventually (if not already) render things that are not Views (where "view" is itself an abstract concept)?

Reply:
Often a Xen object represents a View, and the model for the object (it's `props`) would be the View model. `render` gives one a chance to transmute that model into one specifically designed to interpolate against the template. This is the reason I tend to call it RenderModel and not ViewModel.

The answer to your second question is yes: soon we will have particles that "render" pure data (at that point we may need to change some names).

Reply:
Cool. Thanks for the clarification. `RenderModel` it is.